docs/Metrics-System.md

```diff
@@ -51,20 +51,36 @@ The following shows the details of the available metrics.
 * CapacityTotal: Total capacity of the file system in bytes.
 * CapacityUsed: Used capacity of the file system in bytes.
 * CapacityFree: Free capacity of the file system in bytes.
+* FilesPinned: Total number of the files pinned.
+* PathsTotal: Total number of files and directories in the file system.
 * UnderFsCapacityTotal: Total capacity of the under file system in bytes.
 * UnderFsCapacityUsed: Used capacity of the under file system in bytes.
 * UnderFsCapacityFree: Free capacity of the under file system in bytes.
 * Workers: Number of the workers.
-* FilesTotal: Total number of files and directories in the file system.
-* FilesCheckpointed: Total number of the files checkpointed.
-* FilesCreated: Total number of files and directories created.
-* FilesDeleted: Total number of files and directories deleted.
-* FilesPinned: Total number of the files pinned.
-* FilesRenamed: Total number of file and directories renamed.
+* DirectoriesCreated: Total number of directories created.
+* FileBlockInfosGot: Total number of the file block infos retrieved.
+* FileInfosGot: Total number of the file infos retrieved.
+* FilesCompleted: Total number of files completed.
+* FilesCreated: Total number of files created.
+* FilesFreed: Total number of files freed.
+* FilesPersisted: Total number of the files persisted.
+* NewBlocksRequested: Total number of new blocks requested.
+* PathsDeleted: Total number of files and directories deleted.
+* PathsMounted: Total number of paths mounted.
+* PathsRenamed: Total number of files and directories renamed.
+* PathsUnmounted: Total number of paths unmounted.
+* CreateDirectoryOps: Total number of the CreateDirectory operations.
 * CreateFileOps: Total number of the CreateFile operations.
-* DeleteFileOps: Total number of the DeleteFile operations.
-* GetFileStatusOps: Total number of the getClientFileInfo operations.
-* RenameFileOps: Total number of the RenameFile operations.
+* CompleteFileOps: Total number of CompleteFile operations.
+* DeletePathOps: Total number of the DeletePath operations.
+* FreeFileOps: Total number of FreeFile operations.
+* GetFileBlockInfoOps: Total number of GetFileBlockInfo operations.
+* GetFileInfoOps: Total number of GetFileInfo operations.
+* MountOps: Total number of Mount operations.
+* NewBlockRequestOps: Total number of NewBlockRequest operations.
```

Comment:
I think it would be better to use Get instead of Request and Got instead of Requested for
GetFileInfo, GetFileBlockInfo and GetNewBlock because 'Get' prefix is aligned with methods signature